corehq/apps/locations/models.py

```diff
@@ -352,6 +352,7 @@ class SQLLocation(AdjListModel):
     created_at = models.DateTimeField(auto_now_add=True)
     last_modified = models.DateTimeField(auto_now=True, db_index=True)
     is_archived = models.BooleanField(default=False)
+    archived_at = models.DateTimeField(null=True, blank=True)
```

Comment:
Most event timestamp fields (`created_on`, `received_on`, `closed_on`) are named `_on` rather than `_at`.
```sh
$ ag '_at = ' -s --py | wc -l
      12

$ ag '_on = ' -s --py | wc -l
     238
```

What do you think about changing this one?

Reply:
there is `created_at` right above where i added this field but i see that we use more "_on" in the code base. Will update. thanks for flagging that @millerdev

Reply:
Oof. I hadn't noticed `created_at` ... that's sad. Thanks for renaming.

Reply:
its strange how i named the branch `add-archived-on` and used `archived_at` as the name